CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+* **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened and share more information using the guidelines below.
+
+Provide more context by answering these questions:
+
+* **Did the problem start happening recently** (e.g. after updating to a new version of Mob Engine) or was this always a problem?
+* If the problem started happening recently, **can you reproduce the problem in an older version of Mob Engine?** What's the most recent version in which the problem doesn't happen? You can download older versions of Mob Engine from [the releases page](https://github.com/minetest-mods/mob-engine/releases).
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+
+Include details about your configuration and environment:
+
+* **Which version of Mob Engine are you using?**
+* **Which version of the Minetest you're using?**
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Mob Engine, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
+
+Before creating enhancement suggestions, please check [this list](#before-submitting-an-enhancement-suggestion) as you might find out that you don't need to create one. When you are creating an enhancement suggestion, please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill in [the template](ISSUE_TEMPLATE.md), including the steps that you imagine you would take if the feature you're requesting existed.
+
+#### Before Submitting An Enhancement Suggestion
+
+**Perform a [cursory search](https://github.com/minetest-mods/mob-engine/issues?&q=is%3Aissue)** to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/).Create an issue on that repository and provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://daringfireball.net/projects/markdown/syntax#precode).
```

Comment:
Really? :D
If you make a feature request, you should already add code snippets to make it a good feature request?